Assets/Scripts/Augment/AugmentImplementations/Hat/HatBarrel.cs

```diff
@@ -34,6 +34,8 @@ public class HatBarrel : ProjectileController
     [SerializeField]
     private VisualEffect hatVfx;
 
+    [SerializeField]
+    private float maxLengthBeforeStuck = 100;
```

Comment:
Distance instead of length? And add a newline before Awake?

Reply:
Ye, I agree